index.html

```diff
@@ -1,10 +1,206 @@
 <!DOCTYPE html>
-<html lang="en">
+<html lang="vi">
+
 <head>
-	<meta charset="UTF-8">
-	<title>Document</title>
+  <meta charset="UTF-8">
+  <title>HTML & CSS</title>
+  <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/font-awesome/4.4.0/css/font-awesome.min.css">
+  <link rel="stylesheet" type="text/css" href="css/style.css">
 </head>
+
 <body>
-	<h1>Tìm hiểu html</h1>
+  <div class="wrapper">
+    <!-- header -->
+    <div class="header">
+      <div class="header-content">
+        <div class="logo">
+          <a href="#"><img src="images/logo.png" alt=""></a>
+        </div>
+        <div class="menu">
+          <ul>
+            <li><a href="#">TRANG CHỦ</a></li>
```

Comment:
Sao đoạn text này lại viết in hoa? Viết text thường rồi dùng css để hiển thị text in hoa lên